android/app/src/main/kotlin/com/chicagoroboto/utils/ChicagoRobotoAdapter.kt

```diff
@@ -0,0 +1,55 @@
+package com.chicagoroboto.utils
+
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import android.view.ViewGroup
+import android.view.animation.AnimationUtils
+
+/**
+ * Adapter base class to optionally add animations to the view items
+ */
+open class ChicagoRobotoAdapter<T: RecyclerView.ViewHolder>(private var mLastPosition:Int = -1,
+                                                            var mAnimationType:Int = android.R.anim.slide_in_left,
+                                                            var mAnimationDuration:Long = ANIMATION_DURATION,
+                                                            var mShowAnimation:Boolean = true) :
```

Comment:
Public fields shouldn't hungariant notation. Moreover, no fields _ever_ should use hungarian notation.